ch_backup/util.py

```diff
@@ -87,6 +89,51 @@ def list_dir_files(dir_path: str) -> List[str]:
     ]
 
 
+def scan_dir_files(
+    dir_path: Path, exclude_file_names: Optional[List[str]] = None
+) -> Iterable[str]:
+    """
+    Yields relative file paths in a given directory and excludes files with given names
+    """
+
+    def scan_recursive(dir_path: Path, relative_prefix: Path = None) -> Iterable[str]:
+        with os.scandir(dir_path) as scan:
```

Comment:
It's ok, but why not `os.walk()` ? This approach is just harder to maintain.

Reply:
`os.walk` yields all files and directory names from a given directory at once, so for a directory with a lot of files or other directories returned tuple might be large